examples/framework-remix/README.md

```diff
@@ -6,9 +6,18 @@ This is a [Remix](https://remix.run/) v1 project bootstrapped with [`create-remi
 
 ## Getting Started
 
+Use [`create-next-app`](https://www.npmjs.com/package/create-next-app) with [npm](https://docs.npmjs.com/cli/init), [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/), or [pnpm](https://pnpm.io) to bootstrap the example:
+
+```bash
+npx create-next-app --example https://github.com/inngest/inngest-js/tree/main/examples/framework-remix inngest-remix
```

Comment:
this is kind of weird to use CNA to grapb the remix example, but funny it probably works